src/twyn/dependency_parser/abstract_parser.py

```diff
@@ -17,7 +17,7 @@ class AbstractParser(ABC):
 
     def __init__(self, file_path: str) -> None:
         self.file_path = Path(os.path.abspath(os.path.join(os.getcwd(), file_path)))
-        self.file_handler = FileHandlerPathlib(file_path=self.file_path)
+        self.file_handler = FileHandler(file_path=self.file_path)
```

Comment:
next PR will make this class receive the FileHandler and some other improvements